test/functional/filters_controller_test.rb

```diff
@@ -68,4 +68,23 @@ class FiltersControllerTest < ActionController::TestCase
 
     assert_select "table[data-table='inline']"
   end
+
+  test 'should disable overriding and start inheriting taxonomies from roles' do
+    permission1 = FactoryGirl.create(:permission, :domain, :name => 'permission1')
+    role = FactoryGirl.build(:role, :permissions => [])
+    role.add_permissions! [permission1.name]
+    org1 = FactoryGirl.create(:organization)
+    org2 = FactoryGirl.create(:organization)
+    role.organizations = [ org1 ]
+    role.filters.reload
+    filter_with_org = role.filters.detect { |f| f.allows_organization_filtering? }
```

Comment:
https://speakerdeck.com/sferik/writing-fast-ruby?slide=15 , `.detect(&:allows_organization_filtering?)` should be 20% faster. Not that I want to outsmart the compiler but if somebody already benchmarked it.. We should enable that cop